vocab/properties.json

```diff
@@ -1107,7 +1050,7 @@
         ]
     },
     "laterality": {
-        "description": null,
+        "description": "Dominance in function or use of one of a pair of lateral homologous parts of the body.",
```

Comment:
In my opinion it does, because "function or use" can be interpreted in many different ways. E.g. "function" can mean the left kidney works better than your right. Or the left hemisphere is more responsible for something than the right. "Use" can mean that the left hemisphere was used not the right. And the portion of left vs right hand in handedness reflects the laterality of handedness. So it fits there as well, although we are more specific in handedness now.